commons/src/main/java/br/com/studae/commons/command/annotation/Description.java

```diff
@@ -0,0 +1,13 @@
+package br.com.studae.commons.command.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.PARAMETER})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Description {
+
+    String value();
+}
```

Comment:
Acho que podemos unir as anotações `Name` e `Description` em uma só anotação que recebe ambos os valores e ainda adicionar uma nova opção chamada `required`
```suggestion
package br.com.studae.commons.command.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

@Target({ElementType.PARAMETER})
@Retention(RetentionPolicy.RUNTIME)
public @interface Parameter {

    String name();
    
    String description();
    
    boolean required() default false;
    
}
```